### PR TITLE
Remove Oat Wrapper

### PR DESCRIPTION
**Pull request by dr3**
PartResolves #1176

**Overall change:** Removes oat wrapper, sharing the wrapper used for the headline

**Code changes:**

- Removes the oat wrapper
- Updates the snapshots

This is part 1 of #1176, Remove the oat wrapper first, second to update colour of page background.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [ ] Test engineer approval
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

**Review by ChrisBAshton — DISMISSED** [review]
This ready to review? Because it looks good to me!

**Comment by dr3** [comment]
@ChrisBAshton I think it is, but the issue itself isn't refined so didn't want to put it on the board

**Review by bcmn — DISMISSED** [review]
Checked out locally, looks great (so subtle though 🔭)

File: src/app/lib/styledGrid.js [hunk 1]
```diff
@@ -1,13 +1,10 @@
 import styled from 'styled-components';
-import { C_OAT_LHT } from '@bbc/psammead-styles/colours';
+import { C_GHOST } from '@bbc/psammead-styles/colours';
 import { layoutGridWrapper, layoutGridItemConstrained } from './layoutGrid';
 
 export const Wrapper = styled.div`
```
Comment (sareh) [inline src/app/lib/styledGrid.js]: Could you please call this `GhostWrapper` so that when using it, it's clear that it is ghost-coloured.

**Review by bcmn — APPROVED** [review]
GhostWrapper sounds like everyone's favourite shoestring budget horror movie.

**Review by sareh — APPROVED** [review]
👍 Looks good!

**Comment by bcmn** [comment]
Double checked RGB values with Natalie.

**Pull request merged**